src/extractor/extractColors.ts

```diff
@@ -59,7 +60,8 @@ const extractFills = (paint): fillValuesType | gradientValuesType => {
     return {
       fill: {
         value: convertPaintToRgba(paint),
-        type: 'color' as PropertyType
+        type: 'color' as PropertyType,
+        blendMode: paint.blendMode.toLowerCase()
```

Comment:
Sorry, I missed this. Same needs to be done here `paint.blendMode?.toLowerCase() || 'normal'`